ansible/inventory/hosts.yml

```
@@ -0,0 +1,74 @@
+          ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+          domain_name: "{{ vault_dev_domain_name }}"
+          server_role: development
+      vars:
+        # Development environment settings
+        debug_mode: true
+        enable_ssl: false
+        enable_https_redirect: false
+        
+    staging:
+      hosts:
+        # Add staging servers here when needed
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)